src/bin/pg_autoctl/cli_do_monitor.c

```diff
@@ -116,7 +116,9 @@ static CommandLine monitor_resume_node_command =
 
 static CommandLine monitor_version_command =
 	make_command("version",
-				 "Check and display monitor's extension version",
+				 "Check that monitor version is "
```

Comment:
I expected this change to reflect in the documentation as well, but the PR does not show updated documentation. You might want to update https://github.com/citusdata/pg_auto_failover/blob/d8ba26f47a60eaf1e1dc63ae67279553984d84f5/docs/ref/reference.rst#pg_autoctl-do with this PR as well